javaslang-benchmark/src/test/java/javaslang/benchmark/collection/ArrayBenchmark.java

```diff
@@ -87,39 +85,59 @@ public Object java_mutable(Initialized state) {
         }
 
         @Benchmark
-        public Object slang_persistent(Initialized state) { return state.slangPersistent.tail(); }
+        public Object slang_persistent() { return slangPersistent.tail(); }
     }
 
     public static class Get extends Base {
-        final int index = CONTAINER_SIZE / 2;
-
         @Benchmark
-        public Object java_mutable() { return javaMutable.get(index); }
+        public void java_mutable() {
+            for (int i = 0; i < ELEMENTS.length; i++) {
```

Comment:
it seems more accurate to measure all the `get`s for all possible values, as the compiler seems to inline the value otherwise (for linked lists at least)